css/style.css

```diff
@@ -25,15 +25,15 @@ body {
     font-size: 25px;
     letter-spacing: 2px;
     margin-left: 10px;
-    height: 2vh;
+    height: 15px;
     margin-bottom: 15px;
 }
 
 .credit-card-front .information-wrapper {
     display: flex;
     justify-content: flex-start;
     align-items: center;
-    height: 3.5vh;
+    height: 35px;
```

Comment:
I also corrected this because when zooming in and out, it became a mess. I know understand not to use vh to position things.